SVG/stroke-dasharray/pieChart.xsl

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="3.0"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+    exclude-result-prefixes="xs math">
+    
+    <xsl:output method="xml" indent="yes"/> 
+    
+    <xsl:variable name="KG" select="collection('xml/?select=*.xml')"/>
+    <xsl:variable name="total" select="$KG//page/*[not(self::line)][not(self::description)]! name() => count()"/>
+    
+    <xsl:variable name="radius" select="200"/>
+    
+    <xsl:variable name="color" select="$KG//page/color => count()"/>
```

Comment:
@Yuying-Jin For example, without the collection, you would be reading from the root element of the source document like this:

```xml
 <xsl:variable name="color" select="//page/color => count()"/>
```